### ISSUE TITLE
Branch display order

### ISSUE THREAD

**Issue by AlessandroCalo** [SublimeGit/SublimeGit#126]
After git checkout, the branch list could be displayed without the current branch. (Alternatively, it could be displayed with the current branch at the bottom or in a different section of the list.)
In the image below, I always go from dev to master but have to press the down key first.

Thanks!

![screen shot 2014-03-24 at 10 30 56 am](https://f.cloud.github.com/assets/2083807/2499595/9a77732a-b358-11e3-82ee-c0c85cc69406.png)

**Comment by michaelpedersen** [comment]
Fair point... The current branch should probably always be at the bottom.
